doc/readme/INSTALL.md

```diff
@@ -142,7 +142,7 @@ You need at least 1GB free disk space. If you compile any interface, roughly 4 G
 
 ## Basics <a name="manual-basics"></a>
 Shogun uses [CMake](https://cmake.org/) for its build. The general workflow is now explained.
-For further details on testing etc, see [DEVELOPING.md](DEVELOPING.md).
+For further details on testing etc, see [DEVELOPING.md](https://github.com/shogun-toolbox/shogun/blob/develop/doc/readme/DEVELOPING.md).
```

Comment:
we generally want a relative link here, not to the latest version

Reply:
also it would be master branch (release) ... but since all that is complicated, we prefer the relative link

Reply:
I wan't able to find the relative path. Is it http://shogun.ml/docs/DEVELOPING.md? 

The link doesn't seem working properly. My browser recognises it as a raw file, not HTML. I also reported the issue at the website repository, https://github.com/shogun-toolbox/shogun-web2/issues/42.